src/app/state/coordination.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * Constants representing names of coordination types,
+ * to help prevent typos.
+ */
+export const COORDINATION_TYPES = {
+  DATASET: 'dataset',
+  EMBEDDING_TYPE: 'embeddingType',
+  EMBEDDING_ZOOM: 'embeddingZoom',
+  EMBEDDING_ROTATION: 'embeddingRotation',
+  EMBEDDING_TARGET_X: 'embeddingTargetX',
+  EMBEDDING_TARGET_Y: 'embeddingTargetY',
+  EMBEDDING_TARGET_Z: 'embeddingTargetZ',
+  SPATIAL_ZOOM: 'spatialZoom',
```

Comment:
Can this support fixed zoom offsets? ie, view A is always 4x the zoom of view B?

Reply:
This has not been implemented in this current PR. But in theory, yes, this can support fixed zoom offsets, either a multiple like x4 or an added factor like +4.

I think the easiest approach to implement this would be to link view A and view B to the same zoom level, and then add two new `props` for the `spatial` component called `zoomOffset` and `zoomMultiple`. So both view A and view B would obtain the same zoom value from the coordination space, but view A would always multiply the value by 4 before passing into DeckGL (assuming it has the prop `zoomMultiple = 4`).